tests/test_admin.py

```diff
@@ -1031,3 +1037,61 @@ def test_has_change_permission_returns_false_for_invalid_request(self):
         request = RequestFactory().get("/admin")
         request.user = self.get_superuser()
         self.assertFalse(model_admin.has_change_permission(request))
+
+
+class ListActionsTestCase(CMSTestCase):
+    def setUp(self):
+        self.modeladmin = admin.site._registry[MenuContent]
+
+    def test_preview_link(self):
+        menu = factories.MenuFactory()
```

Comment:
Where possible tests should follow the AAA principle: https://medium.com/@pjbgf/title-testing-code-ocd-and-the-aaa-pattern-df453975ab80

Tests should always have a clear line break between each A, it makes reading tests much, much easier.